lms/services/oauth2_token.py

```diff
@@ -61,7 +61,7 @@ def get(self):
             ) from err
 
 
-def token_store_service_factory(_context, request):
-    return TokenStore(
+def oauth2_token_service_factory(_context, request):
+    return OAuth2TokenService(
```

Comment:
This is totally fine, but I wonder if it's a bit titularly redundant. Presumably most of our services are in separate files from each other so what service it's a factory is for is clear, the fact it's a service factory rather than another kind is also pretty clear. When we refer to this in the config above it's:

 * `lms.services.oauth2_token.oauth2_token_service_factory`

I feel like we are repeating ourselves a lot, where this might do:

 * `lms.services.oauth2_token.factory`

Read backwards it's a factory, for the oauth2_token, service in LMS.

Reply:
Yeah I think they could all just be called `factory()`